editor/layout/index.js

```diff
@@ -17,11 +17,11 @@ import Header from '../header';
 import Sidebar from '../sidebar';
 import TextEditor from '../modes/text-editor';
 import VisualEditor from '../modes/visual-editor';
-import MetaBoxes from '../meta-boxes';
```

Comment:
We should keep this writing `MetaBox` and `meta-box` :)

Reply:
Okay, makes sense. I think `MetaBoxes` + `meta-boxes` would make the most sense, because the area is a collection of many individual metaboxes?